From 7583c4552c987ae1677181ccdedacc464c1bc86d Mon Sep 17 00:00:00 2001
From: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Date: Wed, 13 Sep 2023 14:01:40 +0200
Subject: [PATCH 0517/1141] arm64: dts: st: add clock-cells to syscfg node on
 stm32mp251

Make the syscfg node a clock provider so clock consumers can reach child
clocks through device-tree.

Signed-off-by: Raphael Gallais-Pou <raphael.gallais-pou@foss.st.com>
Change-Id: I550c1188c8063595b05ab645a799696533e84727
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/327475
Reviewed-by: Gabriel FERNANDEZ <gabriel.fernandez@foss.st.com>
Reviewed-by: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yannick FERTRE <yannick.fertre@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Yannick FERTRE <yannick.fertre@foss.st.com>
---
 arch/arm64/boot/dts/st/stm32mp251.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/boot/dts/st/stm32mp251.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp251.dtsi
@@ -2366,6 +2366,7 @@
 			compatible = "st,stm32mp25-syscfg", "syscon";
 			reg = <0x44230000 0x10000>;
 			clocks = <&scmi_clk CK_SCMI_SYSCPU1>;
+			#clock-cells = <1>;
 		};
 
 		tamp: tamp@46010000 {
